src/Extension/FluentExtension.php

```diff
@@ -1055,6 +1055,14 @@ public function LocaleInformation($locale = null)
             $localeObj = Locale::getDefault();
         }
 
+        if (!$locale && !$localeObj) {
+            // There is no default locale, this can happen when if fluent is installed though locales have been setup
+            // This will happen when doing integration unit testing, though can also happen during
+            // regular website operation
+            // This temporary Locale is created to prevent a invalid argument exception in RecordLocale::__construct()
+            $localeObj = Locale::create(['Title' => 'Temp locale']);
```

Comment:
Probably should set the `Locale` db field to the current locale defined in [`i18n::get_locale()`](https://api.silverstripe.org/5/SilverStripe/i18n/i18n.html#method_get_locale) - and don't set a title.

Reply:
Updated